pubspec.yaml

```diff
@@ -5,7 +5,7 @@ description: A Dart package containing re-usable components for making a pub pac
 homepage: https://github.com/dart-lang/pub_server
 
 environment:
-  sdk: '>=1.5.0 <2.0.0'
+  sdk: '>=1.23.0 <2.0.0'
```

Comment:
Since it includes at least one actual fix (in addition to all the formatting and async/await changes) you could consider updating the version here and then publish it.

Reply:
ack